resources/migrations/001_update_migrations.yaml

```diff
@@ -8394,6 +8394,27 @@ databaseChangeLog:
               - column:
                   name: parameter_id
 
+  - changeSet:
+      id: v51.2024-08-27T00:00:00
```

Comment:
In general we don't want `v51...` migration IDs on the `release-x.50.x` branch. If a migration is targeted for a 50 release, it should have a `v50...` ID.

I think the fix here is to change this to a `v50` ID both on master and in this backport PR. You probably also need to add `- onFail: MARK_RAN` to the precondition so that it doesn't blow up if the column already exists in the table.

Sorry I didn't catch this on the original PR — didn't realize it was being backported.

Reply:
Can I just rename a migration ID? What is the consequence of that? The original migration will be reverted automatically by Liquibase, removing the column, followed by the new migration being applied, adding the column back?

Reply:
Changing the migration ID in-place won't cause Liquibase to revert the original migration. It'll just be treated as a new migration. However, if the migration is idempotent, this is fine as long as you have `onFail: MARK_RAN`.

Reply:
https://github.com/metabase/metabase/pull/47517 fixes this.